packages/docusaurus/package.json

```diff
@@ -63,24 +63,24 @@
     "combine-promises": "^1.1.0",
     "commander": "^5.1.0",
     "copy-webpack-plugin": "^11.0.0",
-    "core-js": "^3.25.1",
+    "core-js": "^3.26.0",
     "css-loader": "^6.7.1",
-    "css-minimizer-webpack-plugin": "^4.0.0",
+    "css-minimizer-webpack-plugin": "^4.2.2",
     "cssnano": "^5.1.13",
     "del": "^6.1.1",
-    "detect-port": "^1.3.0",
+    "detect-port": "^1.5.1",
     "escape-html": "^1.0.3",
     "eta": "^1.12.3",
     "file-loader": "^6.2.0",
     "fs-extra": "^10.1.0",
-    "html-minifier-terser": "^6.1.0",
+    "html-minifier-terser": "^7.0.0",
```

Comment:
same, what are the breaking changes of this new version? How does it affect us? Again I'd rather stick to 6.x particularly if I need to backport this PR => avoid risky changes for PRs to backport